ngraph/core/include/ngraph/pass/low_latency.hpp

```diff
@@ -48,5 +49,49 @@ namespace ngraph
             NGRAPH_RTTI_DECLARATION;
             LowLatency();
         };
+
+        /**
+         * @brief The transformation finds all TensorIterator/Loop layers in the network,
+         * processes all back edges that describe a connection between Result and Parameter
+         * of the TensorIterator/Loop bodies,and inserts ReadValue and Assign layers at the
+         * input and output corresponding to this back edge.
+         * Supported platforms: CPU, GNA.
+         *
+         * The example below describes the changes made by the transformation
+         *  [] - TensorIterator body
+         *  () - new layer
+         *  BE - back-edge
+         *
+         *  before applying the transformation:
+         *  -> input1[BE_1 -> Parameter -> Layers ... -> Result  -> BE_1 ]output1->
+         *
+         *  after applying the transformation:
+         *  ->(ReadValue)-> input1[BE_1 ->Parameter->Layers ...->Result->BE_1]output1 ->(Assign)
+         *                                                                      \
+         *                                                                       ->...
+         * After applying the transformation, the resulting network can be inferred
+         * step by step, the states will store between inferences.
+         */
+        class NGRAPH_API LowLatency_v2 : public ngraph::pass::FunctionPass
```

Comment:
Why did we decide to have LowLatency as a public transformation?
Should we deprecate the V1 transformation?

Reply:
deprecated in ngraph and IE